internal/server/flash.go

```diff
@@ -0,0 +1,58 @@
+package server
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+const flashCookieName string = "flash-lpa-frontend"
+
+type FlashNotification struct {
+	Title       string `json:"name"`
+	Description string `json:"description"`
+}
+
+func SetFlash(w http.ResponseWriter, notification FlashNotification) {
+	str, err := json.Marshal(&notification)
+	if err != nil {
+		return
+	}
+
+	c := &http.Cookie{
+		Name:     flashCookieName,
+		Value:    base64.URLEncoding.EncodeToString(str),
+		HttpOnly: true,
+	}
```

Comment:
Snyk is right to report that this should have `Secure: true`. However, that causes all the Cypress tests to fail because it ignores secure cookies in insecure (HTTP) contexts. Since the secure-ness of warning messages isn't particularly important, I've removed the secure flag until we can find a workaround for Cypress.